tests/unit/Bridge/Symfony/Console/RunCommandJobLauncherTest.php

```diff
@@ -20,7 +24,37 @@ public function testLaunch(): void
         $commandRunner->runAsync('yokai:batch:run', 'test.log', $arguments)
             ->shouldBeCalledTimes(1);
 
-        $launcher = new RunCommandJobLauncher(new JobExecutionFactory(), $commandRunner->reveal(), 'test.log');
+        /** @var JobExecutionStorageInterface|ObjectProphecy $storage */
+        $storage = $this->prophesize(JobExecutionStorageInterface::class);
+
+        $storage->store(
+            Argument::that(
+                function ($jobExecution) use ($config) {
+                    if (!$jobExecution instanceof JobExecution) {
+                        return false;
+                    }
+                    if ($jobExecution->getJobName() !== 'testing') {
+                        return false;
+                    }
+                    if ($jobExecution->getId() !== $config['_id']) {
+                        return false;
+                    }
+                    if ($jobExecution->getStatus()
+                            ->getValue() !== BatchStatus::PENDING) {
```

Comment:
You can also use the `is` function of the BatchStatus class : https://github.com/yokai-php/batch/blob/master/src/BatchStatus.php#L57